app/build.gradle

```diff
@@ -121,6 +133,7 @@ dependencies {
 }
 
 kapt {
+    correctErrorTypes = true
```

Comment:
This should be moved to an own commit so it can be seen why this change was done.